src/components/styles.ts

```diff
@@ -0,0 +1,27 @@
+export const MainContainer = {
+  with: 'calc(100vw - 284px)', 
+  width: '100%',
+  height: '100vh',
+  position: 'relative' as 'relative',
```

Comment:
I'm not an css expert, can you elaborate what's the meaning here?

Reply:
Since we are using mantine package for UI it throws type error if you don't define styles directly in the style property